paper/paper.md

```diff
@@ -1,158 +1,74 @@
+  - FAIR
 authors:
-  - name: First Author
+  - name: Stefan Dvoretskii
     affiliation: 1
-  - name: Last Author
-    orcid: 0000-0000-0000-0000
-    affiliation: 2
+  - name: Philipp Schader
+    orcid: 0000-0002-6075-0757
+    affiliation: 1, 2
+  - name: Marco Nolden
+    orcid: 0000-0001-9629-0564
+    affiliation: 1, 2, 3
+  - name: Josh Moore
+    orcid: 0000-0003-4028-811X
+    affiliation: 4
 affiliations:
-  - name: First Affiliation
+  - name: DKFZ Division of Medical Computing
     index: 1
-  - name: Second Affiliation
+  - name: Helmholtz Metadata Collaboration (HMC) Hub Health, DKFZ, Heidelberg, Germany
     index: 2
-date: 8 November 2023
+  - name: Pattern Analysis and Learning Group, Heidelberg University Hospital
+    index: 3
+  - name: German BioImaging e.V., University of Konstanz, Germany
+    index: 4
+date: 9 October 2024
 cito-bibliography: paper.bib
-event: BH23EU
-biohackathon_name: "BioHackathon Europe 2023"
-biohackathon_url:   "https://biohackathon-europe.org/"
-biohackathon_location: "Barcelona, Spain, 2023"
-group: Project 26
+event: SWAT4HCLS24
+biohackathon_name: "SWAT4HCLS"
+biohackathon_url:   "[https://biohackathon-europe.org/](https://www.swat4ls.org/workshops/leiden2024/hackathon/)"
+biohackathon_location: "Leiden, The Netherlands, 2024"
+group: Group 7
 # URL to project git repo --- should contain the actual paper.md:
-git_url: https://github.com/biohackrxiv/publication-template
+git_url: https://github.com/biohackathon-swat4hcls/swat4hcls-2024--publishing-fair-datasets-from-bioimaging-repositories.git
 # This is the short authors description that is used at the
 # bottom of the generated paper (typically the first two authors):
-authors_short: First Author \emph{et al.}
+authors_short: S. Dvoretskii, P. Schader, M. Nolden, J. Moore
 ---
 
 
-# Introduction
+# **Introduction**
 
-As part of the BioHackathon Europe 2023, we here report...
+During the SWAT4HCLS 2024 hackathon, we formed a group of participants from various research initiatives and consortia in Bioimaging and Medical Imaging, like the German Biobanking e.V, the Helmholtz Metadata Collaboration (HMC) Hub Health, the Division of Medical Image Computing at the German Cancer Research Center (DKFZ) and the EU Horizon project CCE\_DART (Data Rich Clinical Trials).
 
+With 2024 being the 10-year anniversary of FAIR data, our discussion quickly came to revolve around the state of FAIR access to Bioimaging data, and ideas how to improve it. Within the research bioimaging community, the Open Microscopy Environment (OME) Consortium has been working to standardize access to large-scale imaging data. The OME consortium is a long-term effort which has produced rich software and standards (\>=2000). Imaging Data Resource (IDR) has appeared as an annotation rich imaging portal in 2016\. As of today, it provides access to 129 Studies with more than 400TB data in more that 14 million images, continuously growing. (\[@usesMethodIn:Williams2017\]). 
```

Comment:
"the 10-year anniversary" yeah?

Reply:
`(\>=2000)` I'm not sure what this refers to.

Reply:
Thanks for noticing this. I am also not sure what it refers too, I would just remove `(\>=2000)` for now @nolden